src/style.css

```diff
@@ -0,0 +1,152 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.searchContainer--big {
+  box-sizing: border-box;
+  margin: 20px 8px 0;
+  height: 70px;
+  min-width: 200px;
+  background-color: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchContainer--small {
+  box-sizing: border-box;
+  margin: 20px 8px 0;
+  height: 42px;
+  min-width: 200px;
+  background-color: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+}
+
+.searchButton--big {
+  height: 19px;
+  line-height: 19px;
+  margin: 24px 17px 0 25px;
+  background-color: #fff0;
+  border: none;
+  float: left;
+}
+
+.searchButton--small {
+  height: 11px;
+  line-height: 11px;
+  margin: 14px 9px 0 12px;
+  background-color: #fff0;
+  border: none;
+  float: left;
+}
+
+.searchIcon--big {
+  width: 19px;
+  height: 19px;
+}
+
+.searchIcon--small {
+  width: 11px;
+  height: 11px;
+}
+
+.searchContainer--big:hover,
+.searchContainer--small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchContainer--big:focus-within {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.searchContainer--small:focus-within {
+  box-shadow: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.inputWrapper {
+  overflow: hidden;
+}
+
+.searchInput--big {
+  width: 100%;
+  border: none;
+  outline: none;
+  background-color: #fff0;
+  font-family: Avenir, sans-serif;
```

Comment:
move it to `html` selector

Reply:
Simply html selector alone  will not do the job. 
Do you mean  html *  ? This works.

Reply:
ok, moved to html *